handlers/graph_unused_test.go

```diff
@@ -0,0 +1,185 @@
+package handlers
+
+import (
+	"testing"
+
+	"k8s.io/api/apps/v1beta1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph/tree"
+	"github.com/stretchr/testify/assert"
```

Comment:
trivial - this third party import is organized incorrectly - put it up in the third-party section